src/projection/fit.js

```diff
@@ -0,0 +1,48 @@
+import {default as geoStream} from "../stream";
+import boundsStream from "../path/bounds";
+
+function fit(project, extent, object) {
+
+  var w = extent[1][0] - extent[0][0],
+      h = extent[1][1] - extent[0][1],
+      precision = project.precision(),
+      clip = project.clipExtent && project.clipExtent();
+
+  project.scale(1)
+      .translate([0, 0]);
+
+  if (clip !== undefined) {
```

Comment:
The reason I specified `undefined` is that albersUsa doesn't implement clipExtent, so trying to set it would throw an error (not sure whether this is also true of other extended projections).